streampark-console/streampark-console-service/src/main/java/org/apache/streampark/console/system/service/impl/MenuServiceImpl.java

```diff
@@ -133,7 +133,9 @@ public void updateMenu(Menu menu) throws Exception {
     public void deleteMenus(String[] menuIds) throws Exception {
         for (String menuId : menuIds) {
             // Find users associated with these menus/buttons
-            this.baseMapper.deleteRoleMenuByMenuId(Long.parseLong(menuId));
+            LambdaQueryWrapper<Menu> queryWrapper = new LambdaQueryWrapper<>();
+            queryWrapper.eq(StringUtils.isNotEmpty(menuId), Menu::getMenuId, Long.parseLong(menuId));
+            this.remove(queryWrapper);
```

Comment:
There is a bug here,It should be delete record in table `t_role_menu`, not  `t_menu`

Reply:
I was careless, thanks for the reminder, I want to fix it